index.html

```diff
@@ -607,9 +607,10 @@ <h2>
         </dt>
         <dd>
           <p>
-            Diversity for the purposes of diversity and inclusion, is any
-            dimension that can be used to differentiate groups and people from
-            one another. Dimensions can include but are not limited to:  socio-economic status, sexual orientation, religion, race, racial identity, physical appearance, neurotype, nationality, mental health, language, indigeneity, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age.
+            Diversity is variation across a wide range of identities, lived experiences, abilities, and perspectives.
+          </p>
+          <p>
+            This can include, but is not limited to: socio-economic status, sexual orientation, religion, race, racial identity, physical appearance, neurotype, nationality, mental health, language, <a href="https://www.google.com/search?q=indigeneity+definition&oq=indi&aqs=chrome.0.69i59l3j0i433i512j69i57j46i131i433i512j46i131i199i433i465i512j0i512j46i131i433i512j0i433i512.2396j0j7&sourceid=chrome&ie=UTF-8">indigeneity</a>, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age.
```

Comment:
Having read the record of the meeting at https://www.w3.org/2023/08/29-pwe-minutes.html I would +1 changing "race" to "ascribed race".

Reply:
I don't think it's appropriate for the link for "indigeneity" to be a google search.  Please change it to link to a dictionary definition.

Reply:
I might suggest https://www.merriam-webster.com/dictionary/indigenous.  If you do want to stick with the Google search, remove the extraneous stuff:  https://www.google.com/search?q=indigeneity.

Reply:
Now that the extraneous stuff has been removed from the google search URL I can see more clearly what the URL is doing.   And it clearly is NOT a link to a definition.  It is a plain vanilla search that happens to currently show a definition of the term.  That could change any moment if google decided to change the design of its search results page.

I think it would be much better to either link to an actual dictionary definition (that is accessible without a paywall) or include our own definition in the glossary.  We owe it to our readers to define our jargon or point to a stable, accessible definition.

Reply:
I tried to find a better link to suggest, but did not find "indigeneity" in any dictionary other than Oxford Online, which has a paywall.  But I did notice that "indigenousness" appears in all major dictionaries, and as far as I can tell, it means the same thing: the quality of being indigenous.   Indeed, if "indigeneity" was intended to mean something subtly different than "the quality of being indigenous" then I think that would be a problem because our documents should be for a broad audience.   I also see that there is [significant debate](https://journals.sagepub.com/doi/abs/10.1177/1177180119828380) about the meaning of "indigeneity" in the academic literature, presumably because of attempts to define the term in specialized ways in legal contexts.

Based on this research, I suggest changing "indigeneity" to "indigenousness".  As far as I can tell, that would still convey the meaning that we are trying to convey, without using a contested term that is not well established in English.